src/main/fc/config.c

```diff
@@ -176,12 +176,13 @@ static void resetPidProfile(pidProfile_t *pidProfile)
     pidProfile->dterm_notch_cutoff = 160;
     pidProfile->vbatPidCompensation = 0;
     pidProfile->pidAtMinThrottle = PID_STABILISATION_ON;
+    pidProfile->max_angle_inclination = 70.0f;    // 70 degrees
 
     // Betaflight PID controller parameters
     pidProfile->setpointRelaxRatio = 30;
     pidProfile->dtermSetpointWeight = 200;
-    pidProfile->yawRateAccelLimit = 220;
-    pidProfile->rateAccelLimit = 0;
+    pidProfile->yawRateAccelLimit = 20.0f;
+    pidProfile->rateAccelLimit = 0.0f;
```

Comment:
It's deg/sec/ms. Its more suitable than huge numbers.

You might be confused due to multiplier in msp code. Thats just to scale it up and down for configurator.